File: Lib/Telerik/Examples/DemoAppsHub/ChartViewExample/ChartViewRotateExampleUserControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms
Imports Telerik.WinControls.UI
Imports Telerik.Charting

Namespace DemoAppsHub.ChartViewExample
	Public Partial Class ChartViewRotateExampleUserControl
		Inherits UserControl
		Public Sub New()
			InitializeComponent()

			Dim barSeries As BarSeries = New BarSeries("Performance", "RepresentativeName")
			barSeries.LegendTitle = "Q1"
			barSeries.Name = "Q1"
			barSeries.DataPoints.Add(New CategoricalDataPoint(177, "Harley"))
			barSeries.DataPoints.Add(New CategoricalDataPoint(128, "White"))
			barSeries.DataPoints.Add(New CategoricalDataPoint(143, "Smith"))
			barSeries.DataPoints.Add(New CategoricalDataPoint(111, "Jones"))
			barSeries.DataPoints.Add(New CategoricalDataPoint(118, "Marshall"))
			Me.radChartView1.Series.Add(barSeries)

			Dim barSeries2 As BarSeries = New BarSeries("Performance", "RepresentativeName")
			barSeries2.LegendTitle = "Q2"
			barSeries2.Name = "Q2"
			barSeries2.DataPoints.Add(New CategoricalDataPoint(153, "Harley"))
			barSeries2.DataPoints.Add(New CategoricalDataPoint(141, "White"))
			barSeries2.DataPoints.Add(New CategoricalDataPoint(130, "Smith"))
			barSeries2.DataPoints.Add(New CategoricalDataPoint(88, "Jones"))
			barSeries2.DataPoints.Add(New CategoricalDataPoint(109, "Marshall"))
			Me.radChartView1.Series.Add(barSeries2)

			Dim barSeries3 As BarSeries = New BarSeries("Performance", "RepresentativeName")
			barSeries3.LegendTitle = "Q3"
			barSeries3.Name = "Q3"
			barSeries3.DataPoints.Add(New CategoricalDataPoint(10, "Harley"))
			barSeries3.DataPoints.Add(New CategoricalDataPoint(55, "White"))
			barSeries3.DataPoints.Add(New CategoricalDataPoint(121, "Smith"))
			barSeries3.DataPoints.Add(New CategoricalDataPoint(86, "Jones"))
			barSeries3.DataPoints.Add(New CategoricalDataPoint(99, "Marshall"))
			Me.radChartView1.Series.Add(barSeries3)

			Me.radChartView1.ShowLegend = True
			Me.radChartView1.LegendTitle = "Performance"

			Me.radChartView1.ShowPanZoom = True
		End Sub
	End Class
End Namespace

File: Lib/Telerik/Examples/DemoAppsHub/ChartViewExample/ChartViewRotateExampleUserControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls.UI;
using Telerik.Charting;

namespace DemoAppsHub.ChartViewExample
{
    public partial class ChartViewRotateExampleUserControl : UserControl
    {
        public ChartViewRotateExampleUserControl()
        {
            InitializeComponent();

            BarSeries barSeries = new BarSeries("Performance", "RepresentativeName");
            barSeries.LegendTitle = "Q1";
            barSeries.Name = "Q1";
            barSeries.DataPoints.Add(new CategoricalDataPoint(177, "Harley"));
            barSeries.DataPoints.Add(new CategoricalDataPoint(128, "White"));
            barSeries.DataPoints.Add(new CategoricalDataPoint(143, "Smith"));
            barSeries.DataPoints.Add(new CategoricalDataPoint(111, "Jones"));
            barSeries.DataPoints.Add(new CategoricalDataPoint(118, "Marshall"));
            this.radChartView1.Series.Add(barSeries);

            BarSeries barSeries2 = new BarSeries("Performance", "RepresentativeName");
            barSeries2.LegendTitle = "Q2";
            barSeries2.Name = "Q2";
            barSeries2.DataPoints.Add(new CategoricalDataPoint(153, "Harley"));
            barSeries2.DataPoints.Add(new CategoricalDataPoint(141, "White"));
            barSeries2.DataPoints.Add(new CategoricalDataPoint(130, "Smith"));
            barSeries2.DataPoints.Add(new CategoricalDataPoint(88, "Jones"));
            barSeries2.DataPoints.Add(new CategoricalDataPoint(109, "Marshall"));
            this.radChartView1.Series.Add(barSeries2);

            BarSeries barSeries3 = new BarSeries("Performance", "RepresentativeName");
            barSeries3.LegendTitle = "Q3";
            barSeries3.Name = "Q3";
            barSeries3.DataPoints.Add(new CategoricalDataPoint(10, "Harley"));
            barSeries3.DataPoints.Add(new CategoricalDataPoint(55, "White"));
            barSeries3.DataPoints.Add(new CategoricalDataPoint(121, "Smith"));
            barSeries3.DataPoints.Add(new CategoricalDataPoint(86, "Jones"));
            barSeries3.DataPoints.Add(new CategoricalDataPoint(99, "Marshall"));
            this.radChartView1.Series.Add(barSeries3);

            this.radChartView1.ShowLegend = true;
            this.radChartView1.LegendTitle = "Performance";

            this.radChartView1.ShowPanZoom = true;
        }
    }
}
